src/main/java/ui/components/pickers/LabelPickerDialog.java

```diff
@@ -202,25 +277,54 @@ private Tooltip createTitleTooltip(TurboIssue issue) {
         return titleTooltip;
     }
 
-    private FlowPane createTopPane() {
-        FlowPane topPane = new FlowPane();
-        topPane.setPadding(new Insets(20, 0, 10, 0));
-        topPane.setHgap(5);
-        topPane.setVgap(5);
-        return topPane;
+    private void setTitleLabel(TurboIssue issue) {
+        title.setText((issue.isPullRequest() ? "PR #" : "Issue #")
+                + issue.getId() + ": " + issue.getTitle());
+    }
+
+    private Label createTextLabel(String input) {
+        Label label = new Label(input);
+        label.setPadding(new Insets(2, 5, 2, 5));
+        return label;
+    }
+
+
+    private Label createGroupTitle(String name) {
+        Label groupName = new Label(name);
+        groupName.setPadding(new Insets(0, 5, 5, 0));
+        groupName.setMaxWidth(ELEMENT_MAX_WIDTH - 10);
+        groupName.setStyle("-fx-font-size: 110%; -fx-font-weight: bold; ");
+        return groupName;
     }
 
-    private TextField createTextField() {
-        TextField textField = new TextField();
-        textField.setId("labelPickerTextField");
-        textField.setPrefColumnCount(30);
-        return textField;
+    private FlowPane createGroupPane(Insets padding) {
+        FlowPane group = new FlowPane();
+        group.setHgap(5);
+        group.setVgap(5);
+        group.setPadding(padding);
+        return group;
     }
 
-    private VBox createBottomBox() {
-        VBox bottomBox = new VBox();
-        bottomBox.setPadding(new Insets(10, 0, 0, 0));
-        return bottomBox;
+    // Event handling 
+
+    /**
+     * Updates state of the label picker based on the entire query
+     * NOPMD : Parameters are unused because the entire query is taken into consideration when 
+     * updating the state
+     */
+    private final void handleUserInput(ObservableValue<? extends String> observable, // NOPMD
+                                       String oldText, String newText) {             // NOPMD
```

Comment:
We can apply the usual line wrapping rules. In the worst case, it can be like this:
```java
private final void handleUserInput(
        //reason for NOPMD
        ObservableValue<? extends String> observable, // NOPMD
        //reason for NOPMD
        String oldText, // NO PMD
        String newText) {
```
In the current version it is not clear which two parameters cause the warnings